src/module/system/text-editor.ts

```diff
@@ -534,6 +536,7 @@ class TextEditorPF2e extends TextEditor {
 
         const item = args.rollData?.item instanceof ItemPF2e ? args.rollData?.item : null;
         const actor = (args.rollData?.actor instanceof ActorPF2e ? args.rollData?.actor : null) ?? item?.actor ?? null;
+        const domains = params.domains?.split(",").filter((d) => sluggify(d) === d && d.endsWith("-damage"));
```

Comment:
```suggestion
        const domains = params.domains?.split(",")..map((d) => sluggify(d)).filter((d) =>  /^[a-z][-a-z0-9]+-damage$/.test(d));
```

Reply:
We can do the regex but I think if its not a valid slug, it should fail rather than surprise anyone.